packages/ui/src/composables/useSelectable.ts

```diff
@@ -62,11 +62,29 @@ export const useSelectable = (
 ) => {
   checkDuplicates(props)
 
+  const isValueReset = ref(false)
+
   /** @public */
-  const reset = () => emit('update:modelValue', false)
-  const focus = () => input.value?.focus()
+  const reset = () => {
+    isValueReset.value = true
```

Comment:
This looks overall confusing for me. We use reset method in useValidation. How about handle this issue in scope of this method? I mean, if value changed with `resetValidation` method from `useValidation`, then we skip validation.

This way you will not need to pass additional options, etc.

Reply:
Reset validation in the `useValidation` composable resets only validation, not the component.